dashboard/lti_new.py

```diff
@@ -232,6 +232,11 @@ def extract_launch_variables_for_tool_use(request, message_launch):
     # Add user to DB if not there; avoids Django redirection to login page
     try:
         user_obj = User.objects.get(username=username)
+        # update
+        user_obj.first_name = first_name
+        user_obj.last_name = last_name
+        user_obj.email = email
+        user_obj.save()
```

Comment:
What is the purpose of save this to user object since it will be populated from Cron job from respective record store. Do you see any difference in user name information from record store and LTI?

Reply:
This is the `auth_user` (logged in user) info. I added this since it is currently never updated after create (and its normal to update this kind of info when LTI launching into the app).

now your absolutely right that the user's name is also in the `user` table but I figured that it would be better to focus on the `auth_user` table since its one-to-one for the current user and the `user` table is technically one-to-many (since the user table is more of an flattened `user`+`enrollment`+`overall_score` table)